site/content/en/docs/Installation/Terraform/gke.md

```diff
@@ -0,0 +1,135 @@
+1. Create or select a project.
+1. Wait for the API and related services to be enabled. This can take several minutes.
+1. [Enable billing][billing] for your project.
+  * If you are not an existing GCP user, you may be able to enroll for a $300 US [Free Trial][trial] credit.
+
+[kubernetes]: https://console.cloud.google.com/kubernetes/list
+[billing]: https://support.google.com/cloud/answer/6293499#enable-billing
+[trial]: https://cloud.google.com/free/
+
+### Choosing a shell
+
+To complete this quickstart, we can use either [Google Cloud Shell][cloud-shell] or a local shell.
+
+Google Cloud Shell is a shell environment for managing resources hosted on Google Cloud Platform (GCP). Cloud Shell comes preinstalled with the [gcloud][gcloud] and [kubectl][kubectl] command-line tools. `gcloud` provides the primary command-line interface for GCP, and `kubectl` provides the command-line interface for running commands against Kubernetes clusters.
+
+If you prefer using your local shell, you must install the gcloud and kubectl command-line tools in your environment.
+
+[cloud-shell]: https://cloud.google.com/shell/
+[gcloud]: https://cloud.google.com/sdk/gcloud/
+[kubectl]: https://kubernetes.io/docs/user-guide/kubectl-overview/
+
+#### Cloud shell
+
+To launch Cloud Shell, perform the following steps:
+
+1. Go to [Google Cloud Platform Console][cloud]
+1. From the top-right corner of the console, click the 
+   **Activate Google Cloud Shell** button: ![cloud shell](../../../../images/cloud-shell.png)
+1. A Cloud Shell session opens inside a frame at the bottom of the console. Use this shell to run `gcloud` and `kubectl` commands.
+1. Set a compute zone in your geographical region with the following command. The compute zone will be something like `us-west1-a`. A full list can be found [here][zones].
+   ```bash
+   gcloud config set compute/zone [COMPUTE_ZONE]
+   ```
+
+[cloud]: https://console.cloud.google.com/home/dashboard
+[zones]: https://cloud.google.com/compute/docs/regions-zones/#available
+
+#### Local shell
+
+To install `gcloud` and `kubectl`, perform the following steps:
+
+1. [Install the Google Cloud SDK][gcloud-install], which includes the `gcloud` command-line tool.
+1. Initialize some default configuration by running the following command.
+   * When asked `Do you want to configure a default Compute Region and Zone? (Y/n)?`, enter `Y` and choose a zone in your geographical region of choice.
+   ```bash
+   gcloud init
+   ```
+1. Install the `kubectl` command-line tool by running the following command:
+   ```bash
+   gcloud components install kubectl
+   ```
+
+[gcloud-install]: https://cloud.google.com/sdk/docs/quickstarts
+
+## Installation
+
+The first step is to enable the `Kubernetes Engine API`. From the Cloud Console, navigate to APIs & Services > Dashboard, then click `Enable APIs and Services`. Type `kubernetes` in the search box to find the Kubernetes Engine API. Click Enable.
```

Comment:
This part now needs to be taken out, as it's redundant with the section above (by this point, you should already have the api enabled and access to gcloud). 

I'd suggest removing these two paragraphs and the next heading "Example Configuration" and just starting the Installation section with the paragraph "An example configuration can be found here"